3573/0/about_control_statements.rb

```diff
@@ -0,0 +1,144 @@
+# rubocop:disable Lint/LiteralAsCondition, Lint/UnneededCopDisableDirective
+# rubocop:disable Metrics/ClassLength, Metrics/MethodLength
+
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+# :reek:FeatureEnvy
+# :reek:RepeatedConditional
+# :reek:UncommunicativeVariableName
+# :reek:TooManyStatements
+class AboutControlStatements < Neo::Koan
+  def test_if_then_else_statements
+    result = if true
+               :true_value
+             else
+               :false_value
+             end
+    assert_equal :true_value, result
+  end
+
+  def test_if_then_statements
+    result = :default_value
+    result = :true_value if true
+    assert_equal :true_value, result
+  end
+
+  def test_if_statements_return_values
+    value = if true
+              :true_value
+            else
+              :false_value
+            end
+    assert_equal :true_value, value
+
+    value = if false
+              :true_value
+            else
+              :false_value
+            end
+    assert_equal :false_value, value
+
+    # NOTE: Actually, EVERY statement in Ruby will return a value, not
+    # just if statements.
+  end
+
+  def test_if_statements_with_no_else_with_false_condition_return_value
+    value = (:true_value if false)
+    assert_equal nil, value
+  end
+
+  def test_condition_operators
+    assert_equal :true_value, (true ? :true_value : :false_value)
+    assert_equal :false_value, (false ? :true_value : :false_value)
+  end
+# rubocop:disable all
```

Comment:
Don't use such comment, try to resolve as many errors as you can, and disable only errors which you can't